Add wrapper projects of various frameworks

Matsuuu opened the pull request.

pshihn commented:
This looks good. Why have two WC projects?
Can you pick one that you prefer. I think both are yours.

Matsuuu commented:
I made the lit one first, and then decided that this didn't need such level of complexity. 
I think I'll rebrand the vanilla version as `wc-rough-notation` or just `<rough-notation>`, and will update here with the updated list :) 
Will do it tomorrow and mention it here!

Matsuuu commented:
Hey @pshihn . 

I removed the lit element one from the list, and updated the vanilla WC one to use tags `<rough-notation` and `<rough-notation-group>` .

This PR is mergeable now, if you approve of the collection of wrappers

fspoettel commented on README.md:
```diff
@@ -109,3 +109,15 @@ Hides the annotation if showing. This is not animated.
+
+Rough Notation has wrappers for multiple libraries and frameworks
+
+[React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
```
This is a bit hard to read, what about using a bulleted list?

- [React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
- [Svelte Rough Notation](https://github.com/dimfeld/svelte-rough-notation)
It is easier to scan it at a glance.

fspoettel commented on README.md:
```diff
@@ -109,3 +109,15 @@ Hides the annotation if showing. This is not animated.
+
+## Wrappers
+
+Rough Notation has wrappers for multiple libraries and frameworks
```
Missing a `:` at the end

Matsuuu commented:
> Thanks for compiling this and your work on the web components wrapper! Two suggestions after looking at the rendered readme

Oh, I was a bit hasty on those ones and forgot the dashes, or my linter just dropped them for some reason.

Fixed them now @fspoettel . Thanks for spotting them out!

pshihn commented:
Thanks!
